packages/nx/src/config/nx-json.ts

```diff
@@ -192,6 +192,27 @@ export interface NxJsonConfiguration<T = '*' | string[]> {
    * **ALPHA**: Configuration for `nx release` (versioning and publishing of applications and libraries)
    */
   release?: NxReleaseConfiguration;
+
+  /**
+   * If specified Nx will use nx-cloud by default with the given token.
+   * To use a different runner that accepts an access token, define it in {@link tasksRunnerOptions}
+   */
+  accessToken?: string;
```

Comment:
What do you think about `nxCloudAccessToken` or `cloudAccessToken` instead?

I don't think `accessToken` by itself at the root is self-documenting enough for someone who isn't intimately familiar enough with our ecosystem

Reply:
Happy to chat about it, I think its fine though as we will update the schemas to have intellisense and the docs will also show it.

Reply:
I'm okay with `nxCloudAccessToken` but I'll defer to @vsavkin .

Reply:
Victor is as well. Changing it.